{"source": "exploit-database", "cves": ["CVE-2016-3115"], "cpes": {"cpe:/o:redhat:enterprise_linux:7.0:ga:server": {"scores": {"stride": "090900"}}}, "filename": "platforms/multiple/remote/39569.py"}